### ISSUE TITLE
Render substation=transition

### ISSUE THREAD

**Issue by Gazer75** [openinframap/openinframap#138]
There are quite a few of these outside underground hydro power stations in Norway that are not rendered.
They usually have both disconnectors and circuit breakers.
At Blåfalli 3 there are two cable sets coming out to the substation from the two 50MW generator transformers.
https://openinframap.org/#17.9/59.871517/6.073459
A bit south is the transition out from Blåfalli Vik power station.
https://openinframap.org/#17.9/59.846375/5.986945
Årøy hydro power station transition:
https://openinframap.org/#17.9/61.276476/7.15497

There are several 132kV transitions similar to Årøy and Blåfalli Vik around Norway as well.
Basically a fenced in area with the basic switchgear mentioned above.

Power stations like these often have a "substation" inside near the generator hall to supply the local regional and/or distribution grid, so there is no need for a full substation outside.

The we have the 3 cable transition stations along Oslofjord that are quite big as they provide for 7 cables per line (2 per phase + 1 reserve)
https://openinframap.org/#14.62/59.5836/10.63302
https://openinframap.org/#13.17/59.35707/10.57195

Opening this in an editor and using Norway Orthophoto layer you'll see these are not insignificant areas.

**Comment by russss** [comment]
At some point it looks like I broke rendering of these transition substations at high zooms. I've fixed this now.

Note that the [wiki page for substation=transition](https://wiki.openstreetmap.org/wiki/Tag:substation%3Dtransition) says:

> This is not a real substation but a fenced-off area with cable terminals where underground cables connect to an overhead power line. It comprises no active components such as switches or transformers.

If there are switches in the compound I'd argue that it shouldn't be tagged as `substation=transition` based on this.
